js/merged-embeded.json

```diff
@@ -0,0 +1,27 @@
+[
```

Comment:
This file should be named _merged-files.json_ instead.

Reply:
renamed